tests/foreman/cli/test_registration.py

```diff
@@ -278,3 +278,38 @@ def test_positive_custom_facts_for_host_registration(
     for interface in interfaces:
         for interface_name in interface.values():
             assert interface_name in str(host_info['network-interfaces'])
+
+
+@pytest.mark.tier1
+def test_positive_set_host_owner_correctly(
+    rhel8_contenthost,
+    module_target_sat,
+    module_sca_manifest_org,
+    module_location,
+    module_activation_key,
+):
+    """Ensures the host owner name is assigned correctly during the registration process.
+
+    :id: fa3f4e1c-e72b-4a9c-8bd6-bf47a35e9f8a
+
+    :steps:
+        1. Register the host.
+        2. Check the host is registered and verify owner name
```

Comment:
I'd suggest to check this scenario end to end, like verify when host is created/registered with non-admin user and same user is assigned as host-owner, wdyt?

Reply:
I don't know what you mean by end-to-end, but I think this can be part of another test that registers a host, you just add assert.

Reply:
I added an assertion to the existing test instead of creating a new one. Please take a look.